dnf/base.py

```diff
@@ -1631,7 +1634,8 @@ def distro_sync(self, pkg_spec=None):
         if pkg_spec is None:
             self._goal.distupgrade_all()
         else:
-            sltrs = dnf.subject.Subject(pkg_spec)._get_best_selectors(self.sack)
+            sltrs = dnf.subject.Subject(pkg_spec) \
```

Comment:
Actually it would not be better, right now, you don't have to look up to know to which method arguments belong. And you clearly know it is a method of some class because of the `.`.